install-wrappers.sh

```diff
@@ -58,6 +64,45 @@ else
     esac
 fi
 
+if [ -n "${HOST_CLANG-}" ]; then
+    HOST_CLANG_EXE=$(command -v $HOST_CLANG)
+    HOST_CLANG_VER=$(echo "__clang_major__ __clang_minor__ __clang_patchlevel__" | $HOST_CLANG_EXE -E -P -x c - | xargs printf '%d.%d.%d')
+
+    mkdir -p $PREFIX/bin
+
+	# ex. /usr/lib/llvm-17/lib/clang/17
+    resdir=$($HOST_CLANG -print-resource-dir)
+	# ex. /usr/lib/llvm-17
+    llvmdir=${resdir%/lib/clang/*}
```

Comment:
The idea is to remove potentially more than one directory level

For the record, I would not use `pwd` at all but `$PWD` (being more standardized), and your construct seems especially weird. if you want the parent directory then always use `dirname`?

Reply:
Sure, that construct probably wasn't the most elegant, using `dirname` probably is better.

> The idea is to remove potentially more than one directory level

I don't think understand this response here - you mean that the `%/lib/clang/*` wildcard expression here removes potentially many levels under `<base>/lib/clang/many/potential/levels`? Sure, my point is that this could potentially fail to match if e.g. `lib` is something else, and I think `dirname(dirname(clang-exe))` is possibly slightly more robust. But it's not really a big deal either way.